.gitignore

```diff
@@ -0,0 +1,6 @@
+### macOS ###
+## General
```

Comment:
Nit: maybe 
```suggestion
## Apple Finder's file
```

Reply:
If it's more readable for you, I can update it.
@iBYarchitect, what do you think?

Reply:
No difference for me. You can change it.

Reply:
@julyakmita update with your suggestion, please check and resolve the conversation, if everything is good.

Reply:
Thanks!
 LGTM)